src/components/Card/index.tsx

```diff
@@ -342,6 +358,13 @@ const Card: React.FC<CardPropTypes> = ({
           {leftAdornment && <LeftAdornment>{leftAdornment}</LeftAdornment>}
 
           <TextContent hasImage={hasImage}>
+            {period && (
+              <Period>
+                <CalendarAlt size={16} color={color.mars} />
+                <PeriodText color={color.mars}>{period}</PeriodText>
+              </Period>
+            )}
```

Comment:
I wonder if we should have a specific `period` prop for a `generic` component like a card. For a lot of use cases, it doesn't make sense to have a period prop. I think it would be better to have a `header` prop (consistent with the footer one) and use that to display a period label (in our own project, for example, `EventCard`).